src/plugins/intel_gna/src/log/dump.cpp

```diff
@@ -440,11 +440,63 @@ inline void DumpCharArray(std::ostream& dumpFile, const char* carray, size_t cou
     dumpFile << "\n";
 }
 
+class GnaMemStats {
+public:
+    void Add(const std::string memory_tag,
+             const std::string operation_type,
+             const std::string operation_name,
+             const uint32_t size) {
+        if (size == 0)
+            return;
+        const std::pair<std::string, std::string> key =
+            std::make_pair(operation_name, memory_tag + " " + operation_type);
+        if (stats_.count(key) == 0) {
+            stats_[key] = size;
+        } else {
+            stats_[key] += size;
```

Comment:
Can this ever happen for a combination of operation type, number, operand name and memory tag type? Isn't it unique?

Reply:
true